extensions/amp-story/1.0/amp-story.js

```diff
@@ -1064,10 +1064,57 @@ export class AmpStory extends AMP.BaseElement {
       });
     }
 
+    this.buildScreenReaderBackButton_();
+
     // Will resolve when all pages are built.
     return storyLayoutPromise;
   }
 
+  /**
+   * Creates a focusable button for screen readers to navigate back.
+   * @private
+   */
+  buildScreenReaderBackButton_() {
+    const backButton = document.createElement('button');
+    this.element.appendChild(backButton);
+    backButton.classList.add('i-amphtml-story-screen-reader-back-button');
+    backButton.setAttribute('aria-label', 'Previous page');
+
+    backButton.addEventListener('click', () => {
+      this.previous_();
+    });
+
+    this.storeService_.dispatch(
+      Action.SET_ADVANCEMENT_MODE,
+      AdvancementMode.MANUAL_ADVANCE
+    );
```

Comment:
This block should be in the click listener, before you call `this.previous_()` so analytics events get the right advancement mode value.
It is used to know what triggered an advancement, so you'll want to set it for each click.